docs/development/MODES.md

```diff
@@ -0,0 +1,50 @@
+# ZetaClient modes of execution
+ZetaClient can execute in one of three modes:
+    `standard`, `dry`, or `chaos`.
+
+## Standard
+Represents the **standard** mode of execution for the ZetaClient.
+A standard observer-signer
+    observes transactions from ZetaChain,
```

Comment:
Wondering, why we have line breaks like this with indentation? It looks harder to read imo than we just do line break at around the same length as in the `SIMULATION_TESTING` docs, we also want to have some consistencies between docs